dashboard/api_test.py

```diff
@@ -799,3 +803,461 @@ def test_program_no_courses(self, mock_info_course):
             "courses": []
         }
         self.assertEqual(res, expected_data)
+
+
+class CertificatesTests(ESTestCase):
```

Comment:
`CertificatesTests` and `EnrollmentsTests` are 200+ lines each of nearly identical code. if we want to hold off on refactoring this and do it as part of the issue i suggested in my comment on `dashboard/api.py`